src/partials/connectors/connectors-popular.hbs

```diff
@@ -1,134 +1,43 @@
 <section class="connectors-popular">
   <header class="header-connectors-title">Popular Connectors</header>
   <div class="flex popular-cards">
-    <a class="card" href="//docs.mulesoft.com/connectors/salesforce/salesforce-connector">
-      <img
-        alt="Salesforce"
-        class="card-img"
-        height="50"
-        src="{{uiRootPath}}/img/logos/salesforce.png"
-        width="71"
-      >
+    {{#each (array 'salesforce' 'sap' 'amazon-s3') as |connectorName|}}
+    {{#with (lookup @root.site.components (concat connectorName '-connector'))}}
+    {{#with latest.asciidocConfig}}
+    <a class="card" href="{{relativize ../url}}">
+      <img src="{{@root.uiRootPath}}/img/icons/connectors/icon-{{attributes.page-vendor-name}}.svg" alt="{{{attributes.page-vendor-title}}} Logo" class="card-img" width="71" height="50">
       <div>
-        <div class="card-title">Salesforce</div>
-        <div class="card-description">Connect to the Salesforce SOAP API, REST API, Bulk API, and Streaming API</div>
+        <div class="card-title">{{{attributes.page-vendor-title}}}</div>
+        <div class="card-description">{{{attributes.page-component-desc}}}</div>
       </div>
     </a>
-    <a class="card" href="//docs.mulesoft.com/connectors/sap/sap-connector">
-      <img
-        alt="SAP"
-        class="card-img"
-        height="50"
-        src="{{uiRootPath}}/img/logos/sap.png"
-        width="98"
-      >
+    {{/with}}
+    {{else}}
+    <a class="card" href="#">
+      <img src="{{@root.uiRootPath}}/img/icons/connectors/icon-{{connectorName}}.svg" alt="{{connectorName}} Logo" class="card-img" width="71" height="50">
```

Comment:
For the popular connectors, I was thinking the path should be:
`src="{{@root.uiRootPath}}/img/logos/{{attributes.page-vendor-name}}.png"`

Reply:
Also (I'll do this in my PR), I think we'll have to remove setting the width here because the logos will have different widths (but keep the same height).

Reply:
That's fine with me. I just wanted to present you with the various options.